Makefile

```diff
@@ -372,17 +372,17 @@ venv:
 ## Extraction step
 setup:
 	$(MAKE) -C tools
-	$(PYTHON) tools/buildtools/decompress_baserom.py $(VERSION)
+	$(PYTHON) tools/buildtools/decompress_baserom.py --mm-version $(VERSION)
```

Comment:
I'm not sure if I like `--mm-version` for the name of the flag, just `--version` should be fine.

We could even omit the flag entirely since those scripts aren't meant to be invoked by the user directly. Seeing that this script is being passed `$(VERSION)` feels obvious enough, doesn't it?

Reply:
I know there was quite a big disucssion about this in OoT when it was discussed. `--oot-version` was the compromise. Kind of thinking for the sake of keeping things easier to port between games, curious if we should just do something like `--rom-version` or `--game-version` so they could be the same in both.

Reply:
I think I like `--game-version` over `--mm-version`.
@Dragorn421 thoughts?

Reply:
My thoughts are I wanted to avoid -v and --version in the first place as they could be confused with a verbose flag and/or the tool version rather than the game version

However we now have -v so really I have no reason to oppose to anything if my only point is already defeated. We could have --version too

Reply:
I would think it would still be worth it to keep `--version` reserved, but I'm not sure we would even add version info for the script.

Reply:
Personally I would say go with `--game-version` for now.

Reply:
Changed to `-v`,`--version`, not required, and defaults to n64-us